### ISSUE TITLE
module 'plaidml.keras.backend' has no attribute 'normalize_data_format'

### ISSUE THREAD

**Issue by colorfuldarkgray** [plaidml/plaidml#203]
Hi ppl

I'm tryning to run https://www.kaggle.com/keegil/keras-u-net-starter-lb-0-277/notebook

in my plaidml env with python 3.6 and: 
numpy 1.15.4
pandas 0.23.4
keras 2.2.4
tf 1.12.0
plaidml 0.3.4
plaidml-keras 0.3.4

but I get this error: module 'plaidml.keras.backend' has no attribute 'normalize_data_format'

[error.txt](https://github.com/plaidml/plaidml/files/2634748/error.txt)

any help please?

**Comment by brianretford** [comment]
We currently don't support 2.2.4 though we would like to add support soon, as the delta is only a few backend function mods

**Comment by brianretford** [comment]
duplicate of #201
